portal_blocks_documentation/docs.py

```diff
@@ -14,6 +14,9 @@
 BUILD_DIR = project_root / "docs" / "portal_blocks"
 BUILD_DIR.mkdir(exist_ok=True)
 
+MOUNT_DIR = '../content/portal-builder/rules-editor/block-reference'
+
+MENU_FILE = project_root / "hugo" / "themes" / "hugo-geekdoc" / "data" / "menu" / "main.yaml"
```

Comment:
u can just override the `main.yaml`  in `hudo/data` dir.

Reply:
is there a specific benefit for doing it in the hugo dir or in the theme dir? - I would probably open a separate issue for the file placement.

Reply:
Yes, there are various

- this is standard practice in hugo
- theme has a remote repo and changing the theme can lead to conflicts when updating the theme from remote repo